tests/insert_coins.js

```diff
@@ -44,40 +44,41 @@ async function main() {
         const mint = api.tx.mantaPay.toPrivate(mints_buffer.subarray(mints_start, mints_start + mint_size));
         transactions.push(mint);
 
-        const transfers_start = transfers_offset + i * (2 * mint_size + transfer_size);
-        const transfer_mint_1 = api.tx.mantaPay.toPrivate(transfers_buffer.subarray(transfers_start, transfers_start + mint_size));
-        const transfer_mint_2 = api.tx.mantaPay.toPrivate(transfers_buffer.subarray(transfers_start + mint_size, transfers_start + 2 * mint_size));
-        const transfer = api.tx.mantaPay.privateTransfer(transfers_buffer.subarray(transfers_start + 2 * mint_size, transfers_start + 2 * mint_size + transfer_size));
-        transactions.push(transfer_mint_1);
-        transactions.push(transfer_mint_2);
-        transactions.push(transfer);
+        // const transfers_start = transfers_offset + i * (2 * mint_size + transfer_size);
+        // const transfer_mint_1 = api.tx.mantaPay.toPrivate(transfers_buffer.subarray(transfers_start, transfers_start + mint_size));
+        // const transfer_mint_2 = api.tx.mantaPay.toPrivate(transfers_buffer.subarray(transfers_start + mint_size, transfers_start + 2 * mint_size));
+        // const transfer = api.tx.mantaPay.privateTransfer(transfers_buffer.subarray(transfers_start + 2 * mint_size, transfers_start + 2 * mint_size + transfer_size));
+        // transactions.push(transfer_mint_1);
+        // transactions.push(transfer_mint_2);
+        // transactions.push(transfer);
```

Comment:
Why comment out integration test?

Reply:
oh forgot about this. some files were incorrectly added . lemme revert.